games-emulation/bsnes_libretro/bsnes_libretro-0.94_20171227.recipe

```diff
@@ -7,11 +7,12 @@ attempting to provide a reasonable balance between the previous two."
 HOMEPAGE="https://byuu.org/emulation/higan/"
 COPYRIGHT="2014-2018 byuu, the libretro team"
 LICENSE="GNU GPL v3"
-REVISION="1"
+REVISION="2"
 srcGitRev="4e572a3cfa10e56bedb6737b0494e8165f6490d5"
 SOURCE_URI="https://github.com/libretro/bsnes-libretro/archive/$srcGitRev.tar.gz"
 CHECKSUM_SHA256="77255571b654ef1b7559608818393dc8377c657dc30aa7d3ec45e40662c99a31"
 SOURCE_DIR="bsnes-libretro-$srcGitRev"
+SOURCE_FILENAME="bsnes-libretro-${portVersion/_/-}-$srcGitRev.tar.gz"
```

Comment:
```diff
-SOURCE_DIR="bsnes-libretro-$srcGitRev"
 SOURCE_FILENAME="bsnes-libretro-${portVersion/_/-}-$srcGitRev.tar.gz"
+SOURCE_DIR="bsnes-libretro-$srcGitRev"
```
Yep, we prefer to put `SOURCE_DIR` after `SOURCE_FILENAME`. Thanks :)
Source: [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)
(I did some metrics some time ago and found out that there are more recipes with `SOURCE_FILENAME` before `SOURCE_DIR` than the oposite, so we try to stick to the trend.)
(For recipe which already exist, we only swap if we also make a change near that line.)